pkg/artifactory/repositories.go

```diff
@@ -330,426 +332,533 @@ var repoTypesLikeGeneric = []string{
+		},
+		"share_configuration": {
+			Type:     schema.TypeBool,
+			Optional: true,
+			Computed: true,
+		},
+		"synchronize_properties": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+			Description: "When set, remote artifacts are fetched along with their properties.",
+		},
+		"block_mismatching_mime_types": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+			Description: "Before caching an artifact, Artifactory first sends a HEAD request to the remote resource. In some remote resources, HEAD requests are disallowed and therefore rejected, even though downloading the artifact is allowed. When checked, Artifactory will bypass the HEAD request and cache the artifact directly using a GET request.",
+		},
+		"property_sets": {
+			Type:     schema.TypeSet,
+			Elem:     &schema.Schema{Type: schema.TypeString},
+			Set:      schema.HashString,
+			Optional: true,
+		},
+		"allow_any_host_auth": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+			Description: "Also known as 'Lenient Host Authentication', Allow credentials of this repository to be used on requests redirected to any other host.",
+		},
+		"enable_cookie_management": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+			Description: "Enables cookie management if the remote repository uses cookies to manage client state.",
+		},
+		"bypass_head_requests": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+			Description: "Before caching an artifact, Artifactory first sends a HEAD request to the remote resource. In some remote resources, HEAD requests are disallowed and therefore rejected, even though downloading the artifact is allowed. When checked, Artifactory will bypass the HEAD request and cache the artifact directly using a GET request.",
+		},
+		"priority_resolution": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+			Description: "Setting repositories with priority will cause metadata to be merged only from repositories set with this field",
+		},
+		"client_tls_certificate": {
+			Type:     schema.TypeString,
+			Optional: true,
+			Computed: true,
+		},
+		"content_synchronisation": {
+			Type:     schema.TypeList,
+			Optional: true,
+			Computed: true,
+			MaxItems: 1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"enabled": {
+						Type:        schema.TypeBool,
+						Optional:    true,
+						Default:     false,
+						Description: `(Optional) If set, Remote repository proxies a local or remote repository from another instance of Artifactory. Default value is 'false'.`,
+					},
+					"statistics_enabled": {
```

Comment:
It's cool that you added these, but they weren't being used.

Reply:
No, these aren't new. We added them a few weeks ago